cpp-client/README.md

```diff
@@ -44,11 +44,13 @@ C++ compiler and tool suite (cmake etc).
 
    Example:
    ```
-   mkdir -p $HOME/dhcpp
-   cd $HOME/dhcpp
+   export DHCPP=$HOME/dhcpp  # Put your choice of location for the installation here.
+   mkdir -p $DHCPP
+   cd $DHCPP
```

Comment:
Also if you want to be perfectly consistent, way above in line 23, consider changing

~~~
 mkdir -p ~/src/deephaven && cd ~/src/deephaven && export DHSRC=`pwd`
~~~

to
~~~
export DHSRC=~/src/deephaven
mkdir -p $DHSRC
cd $DHSRC
~~~